style.css

```diff
@@ -0,0 +1,177 @@
+*{
+    margin: 0;
+}
+
+body {
+    background-color: #eceef1;
+    font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.header {
+    height: 2.8em;
+    padding: 0.8em 2em;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.header-div {
+    padding: 0.5em;
+    font-size: 12px;
+    color: rgb(69, 69, 69)
+}
+
+
+.icon-button, .submit {
+    background-color: #0077c5;
+    border-radius: 3px;
+}
+
+
+
+.header-div a {
+    color: #0077C5;
+}
+
+.icon-button {
+    border: solid #4c4c4c 1px;
+    padding: 0.5em 0.8em;
+}
+
+.submit {
+    width: auto;
+    padding: 6px 0px 6px 0px;
+    margin-left: 1em;
+    
+}
+
+.icon-button a, .submit a {
+    color: white;
+}
+
+.submit a {
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    height: 2em;
+    align-items: center;
+}
+
+.icon-button a, .header-div a{
+    text-decoration: none;
+}
+
+.footer {
+    text-align: center;
+    font-size: 12px;
+    position: relative;
+    bottom: 1em;
+    width: 100%;
+    margin-top: 2em;
+}
+
+.footer-links {
+    text-decoration: none;
+    color: #505050;
+    padding: 0 0.7em;
+}
+
+.footer-links:hover {
+    text-decoration: underline;
+}
+
+.footer p {
+    color: #505050;
+}
+
+.main-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 50em;
+    flex-direction: column;
+}
+
+.form-content {
+    width: 19.875em;
+    height: 35em;
+    display: flex;
+    flex-direction: column;
+    background-color: #ffffff;
+    border: lightgray solid 1px;
+    border-bottom: none;
+    padding: 1.9em;
+}
+
+.form-title {
+    text-align: center;
+    color: rgb(69, 69, 69)
+}
+
+.form-title h2 {
+    padding: 0.8em 0;
+    font-size: 1.5em;
+    font-weight: lighter;
+}
+
+.form-title p {
+    font-size: smaller;
+}
+
+.form-title a {
+    text-decoration: none;
+    color: #0077C5;
+    font-size: smaller;
+}
+
+.form {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 0.4em;
+}
+
+.containner input {
+    border: solid lightgray 1px;
+    width: 100%;
+    height: 2.3em;
+    border-radius: 3px;
+}
+
+.containner {
+    padding: 1em;
+}
+
+.terms {
+    padding: 1.5em;
+    text-align: center;
+    font-size: smaller;
+    
+}
+
+.terms p {
+    color: rgb(69, 69, 69);
+}
+
+.terms a {
+    color: #0077C5;
+    text-decoration: none;
+}
```

Comment:
This code can be grouped with the code in **lines 174 - 177** as they do the same thing.